teamA/leeHyeonTae/services/userService.js

```diff
@@ -0,0 +1,25 @@
+const userDao = require('../models/userDao');
+
+const signUp = async ( name, email, profile_image, password) => {
+    const pwValidation = new RegExp(
+        '^(?=.*[A-Za-z])(?=.*[0-9])(?=.*[!@#$%^&*_])(?=.{8,20})'
+    );
+    if(!pwValidation.test(password)) {
+        const err = new Error('PassWord is not valid!');
```

Comment:
에러메시지는 클라이언트가 받는 에러 메시지입니다.

여기서 사용한 camelcase는 오히려 가독성이 떨어져 보일 수도 있을 것 같습니다.

간단 명료하지만 내용 전달은 충분히 되는 에러메시지를 작성해보세요!
```suggestion
        const err = new Error('INVALID_PASSWORD');
```